linkml_runtime/utils/schemaview.py

```diff
@@ -111,9 +112,9 @@ class SchemaView(object):
     modifications: int = 0
     uuid: str = None
 
-    def __init__(self, schema: Union[str, SchemaDefinition],
+    def __init__(self, schema: Union[str, PosixPath, SchemaDefinition],
                  importmap: Optional[Mapping[str, str]] = None, merge_imports: bool = False):
-        if isinstance(schema, str):
+        if isinstance(schema, str) or isinstance(schema, PosixPath):
```

Comment:
I had 3 different proposed ways to do this in [this issue](https://github.com/linkml/linkml/issues/1056), but I felt like this way was even simpler than those.